starport/templates/typed/map/stargate.go

```diff
@@ -545,19 +530,7 @@ import "%s/%s.proto";`
 			)
 		}
 
-		// Ensure custom types are imported
-		customFields := append(opts.Fields.Custom(), opts.Indexes.Custom()...)
```

Comment:
why do we remove this code block?

Reply:
May have been a conflict resolution mistake, thanks for spotting this